ontology.ttl

```diff
@@ -861,127 +876,127 @@ Subclasses may have other ways to express the content.""" ;
 ###  http://odahub.io/ontology#ISOT
 :ISOT rdf:type owl:NamedIndividual ,
                :TimeFormat ;
-      :symbol "isot"^^xsd:string .
```

Comment:
why do you propose to remove these?

Reply:
This is a consequence of the change in owlapi library used by Protégé https://github.com/owlcs/owlapi/issues/1063 which is motivated by that the string type is assumed by default.

Reply:
Ok, thanks.